src/tools/EpicsHostArch.pl

```diff
@@ -1,36 +1,109 @@
 #!/usr/bin/env perl
 #*************************************************************************
-# Copyright (c) 2018 UChicago Argonne LLC, as Operator of Argonne
+# Copyright (c) 2023 UChicago Argonne LLC, as Operator of Argonne
```

Comment:
In general you shouldn't modify other people's copyright statements, here the Argonne copyright term started in 2018 when I last modified this file, and the fact that _you_ have made changes to it doesn't affect someone else's start-date. You or your employer may want your own separate copyright line to be added, and that's fine if you do.